src/app/inbox/components/notification-your-record-amended/notification-your-record-amended.component.html

```diff
@@ -1,14 +1,6 @@
-<app-info-drop-down
```

Comment:
Seems like you are removing a required feature displayed here zpl.io/aBJvjqA

Reply:
Ok, I will revert it